.circleci/scripts/download-check-config.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# Downloads and install check_config into /usr/local/bin
+
+set -e
+
+if ! command -v check_config >/dev/null 2>&1; then
+  tempDir=$(mktemp -d)
+  cp ".tool-versions" "$tempDir/"
+
+  pushd "$tempDir" >/dev/null || exit 1
+  REPO=getoutreach/stork
+  TAG=$(gh release -R "$REPO" list | grep Latest | awk '{ print $1 }')
+  echo "Using check_config version: ($TAG)"
+  gh release -R "$REPO" download "$TAG" --pattern "stork_*_$(go env GOOS)_$(go env GOARCH).tar.gz"
+
+  echo "" # Fixes issues with output being corrupted in CI
+  tar xf stork**.tar.gz
+  sudo mv check_config /usr/local/bin/check_config
```

Comment:
Just a suggestion... maybe turn this `/usr/local/bin/check_config` into a reusable variable since we reference it a handful of times.

Reply:
Ah ok, I was just copying `.circleci/scripts/download-stencil` but I will add that in.

Reply:
Just was being nit picky. More just trying to prevent an accidentally typo in the future if we were to change the path or package name.

Reply:
Ah, that makes sense.